app/javascript/crayons/MultiSelectAutocomplete/__tests__/MultiSelectAutocomplete.test.jsx

```diff
@@ -123,25 +123,6 @@ describe('<MultiSelectAutocomplete />', () => {
     ).toBeInTheDocument();
   });
 
-  it('displays search term as an option if no suggestions', async () => {
```

Comment:
This test is no longer required as the functionality of the `<MultiSelectAutocomplete />` has changed